main.go

```diff
@@ -98,6 +99,7 @@ func main() {
 			for {
 				prs, resp, _ := client.Search.Issues(
 					// Search query to find PR's
+					context.Background(),
```

Comment:
This is valid and it would work.

If you wanted to use the context request that's available in this handler, I believe you can simply pass `c` here.

If you look at the signature of the handler:

```Go
// User handler
 router.GET("user/:username", func(c *gin.Context) {
```

From what I can tell, `*gin.Context` implements `context.Context` interface and should work.

I'm not very familiar with "github.com/gin-gonic/gin" package, so I could be off.